README.rst

```diff
@@ -66,7 +66,7 @@ Fiware-Facts
     and publish the result into a RabbitMQ queue to be analysed by Fiware-Cloto. The facts are the result of the server
```

Comment:
It works if you navigate by clicking Section -> Top -> Section -> Top... Github if who is in charge